quickboosters/api/order/dev/sample_data.py

```diff
@@ -15,7 +15,7 @@ def create_order():
   db.session.add(order)
   db.session.commit()
 
-  def order_status():
+def order_status():
     status = Order_Status('status': order_status.enum, 'order_id': order_status.id, 'game': order_status.game)
```

Comment:
Order_Status isn't a class that you create an instance of. You don't need this function actually.  You use enums to represent the status, like this. Order_Status.not_started or Order_status.in_progress. So you use this when you're creating an order object. 

It would look like 
`Orders(order_type="solo_boost", user_id='1', order_amount=69.42, status=Order_Status.in_process, created_date=datetime.datetime.now())`